starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -83,7 +85,11 @@ public Optional<Inventory> update(String id, Inventory inventory) {
    * @return Deleted Inventory.
    */
   public Optional<Inventory> delete(String id) {
-    // TODO
-    return Optional.empty();
+    Query query = Query.query(Criteria.where("id").is(id));
+    Inventory inv = this.mongoTemplate.findOne(query, Inventory.class);
+    if (inv != null) {
+      this.mongoTemplate.remove(inv);
```

Comment:
Is there a better way to search for a delete items from the database with a single DB call rather than 2 separate ones?